### DISCUSSION TITLE
关于JDBC版本返回数据为base64编码的问题

### DISCUSSION THREAD

**Discussion by ChenLiu0921** [taosdata/TDengine#10023] [Q&A]
目前使用的版本为 2.4.04版本  

1. 使用JDBC JNI连接后 (已经设置charset为utf-8 ) show database, show tables 返回的很多字段都为base64编码, 请问是最新版的问题 还是数据库设计就为这样?  使用官方提供的http请求 直接curl服务  返回的数据为正常的utf-8
2. 使用JDBC RESTFUL连接服务   DESC <tablename> 这个命令提示无效  是目前不支持吗?

**Comment by yu285** [comment]
我们的jdbc 驱动是什么版本？

这两个问题可以截图一下么，可以加微信15652223354排查下

**Reply by ChenLiu0921** [reply]
驱动版本用的是 2.0.33

代码中使用的JNI  执行命令show databases  返回的单行数据有的是base64的编码
<img width="437" alt="image" src="https://user-images.githubusercontent.com/37284268/154437889-7bae7f8b-5188-415a-b943-49ef3607052a.png">

**Reply by huolibo** [reply]
怀疑是使用工具做了base64转换，可以写一个demo测试下。
restful支持desc或是asc排序，请列出编写的sql，我们也可以做下测试。
